Core/Utilities/RabbitMQAdmin.py

```diff
@@ -0,0 +1,138 @@
+"""RabbitMQAdmin module serves for the management of the internal RabbitMQ
+   users database. It uses rabbitmqctl command. Only the user with the right
+   permissions can execute those commands.
+"""
+import re
+from DIRAC import S_OK, DError
```

Comment:
DError cannot be imported directly from DIRAC unfortunately, you have to use the full import path :-(